### DISCUSSION TITLE
Voice announcement settings

### DISCUSSION THREAD

**Discussion by 0709wiwiwi** [devemux86/cruiser#553] [Q&A] [answered]
Being able to limit the TTS chatter in Cruiser is blissful. 

_- How to set up Last announcement only ?_ 
 
| Select | Voice guidance/Advanced Navigation instructions | OFF

Couldn't the settings be a bit more simplified ?
OFF:  When the distance gliders are on the far left. 
![Slider left is instruction OFF](https://github.com/devemux86/cruiser/assets/54167201/374eba5b-ac94-4418-b6ca-262b32b4d6d1)

**Comment by devemux86** [comment, marked as answer]
@0709wiwiwi thanks for the suggestion.

This is not so simple, as the voice distance sliders are dynamic.
Their limits are set automatically from the other distance sliders.

Now the distance sliders simply adjust the distances, nothing else.
Switches for on / off usage exist in the advanced section.

**Reply by 0709wiwiwi** [reply]
Having to set Navigation instructions | OFF | in the advanced panel this to keep the Last Navigation instruction activ only is not ideal, imo.  Then try explaining that in the wiki meant for new users.  If not combinable into the distance sliders use on/off select buttons ? 
Note.  
Last announcement as TTS or tone together with display on/off trigger settings may also come one day ?

**Reply by devemux86** [reply]
> Last announcement as TTS or tone together with display on/off trigger settings may also come one day ?

These are already in my list. :slightly_smiling_face:
